File: src/main/java/aurocosh/divinefavor/common/muliblock/validators/AirStateValidator.java
package aurocosh.divinefavor.common.muliblock.validators;

import com.google.gson.annotations.Expose;
import net.minecraft.block.state.IBlockState;
import net.minecraft.init.Blocks;
import vazkii.patchouli.common.multiblock.StateMatcher;

public class AirStateValidator extends StateValidator {
    @Expose
    int test;
    @Override
    public boolean isValid(IBlockState state) {
        return state.getBlock() == Blocks.AIR;
    }

    @Override
    public Object getPatchouliMatcher() {
        return StateMatcher.AIR;
    }
}

File: src/main/java/aurocosh/divinefavor/common/muliblock/validators/AirStateValidator.kt
package aurocosh.divinefavor.common.muliblock.validators

import com.google.gson.annotations.Expose
import net.minecraft.block.state.IBlockState
import net.minecraft.init.Blocks
import vazkii.patchouli.common.multiblock.StateMatcher

class AirStateValidator : StateValidator() {
    @Expose
    internal var test: Int = 0

    override fun isValid(blockState: IBlockState): Boolean {
        return blockState.block === Blocks.AIR
    }

    override fun getPatchouliMatcher(): Any {
        return StateMatcher.AIR
    }
}
